paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -205,8 +205,26 @@ public class PaparazziPlugin @Inject constructor(
           .withPropertyName("paparazzi.nativeRuntime")
           .withPathSensitivity(PathSensitivity.NONE)
 
-        test.outputs.dir(reportOutputDir)
-        test.outputs.dir(snapshotOutputDir)
+        test.inputs.dir(
+          isVerifyRun.flatMap {
+            project.objects.directoryProperty().apply {
+              set(if (it) snapshotOutputDir else null)
+            }
```

Comment:
nit: is the directoryProperty step necessary, vs:

```suggestion
            if (it) snapshotOutputDir else null)
```
?

Otherwise, agreed, I like this better!

Reply:
so the output of `Property.map {}` cant be null so this is a workaround for that.